tests/kfs_fsync_test.c

```diff
@@ -0,0 +1,74 @@
+#include <check.h>
+#include <kfs_open.h>
+#include <kfs_create.h>
+#include <kfs_write.h>
+#include <kfs_fsync.h>
+#include <kfs_flush.h>
+
+#include "test_main.h"
+
+void kfs_fsync_setup(void) {
+    main_setup();
+}
+
+void kfs_fsync_teardown(void) {
+    main_teardown();
+}
+
+START_TEST(kfs_fsync_opened_file) {
+    char dir_path[strlen(LOCAL_DISC_CACHE_PATH) + strlen("opened.txt") + 1];
+    strcpy(dir_path, LOCAL_DISC_CACHE_PATH);
+    char *path = strcat(dir_path, "opened.txt");
+    struct fuse_file_info create = {O_CREAT};
+    int res = kfs_create(path, 0777, &create);
+    fail_if(create.fh == 0);
```

Comment:
not nessesary to check create status all the time, it been tested in create test etc